### PR TITLE
Rename executables to bitcoinz

### PR DESCRIPTION
**Pull request by egyptianbman**
This commit replaces the zcash* executables with bitcoinz* executables.

### CODE REVIEW

File: contrib/bitcoinz-cli.bash-completion [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# bash programmable completion for zcash-cli(1)
-# Copyright (c) 2012-2016 The Bitcoin Core developers
+# bash programmable completion for bitcoinz-cli(1)
+# Copyright (c) 2017-2018 The BitcoinZ developers
```
Comment (renuzit) [inline contrib/bitcoinz-cli.bash-completion]: We must leave the original Copyrights

File: contrib/bitcoinz-tx.bash-completion [hunk 2]
```diff
@@ -1,15 +1,17 @@
-# bash programmable completion for zcash-tx(1)
-# Copyright (c) 2016 The Bitcoin Core developers
+# bash programmable completion for bitcoinz-tx(1)
+# Copyright (c) 2017-2018 The BitcoinZ developers
```
Comment (renuzit) [inline contrib/bitcoinz-tx.bash-completion]: same here

File: contrib/bitcoinzd.bash-completion [hunk 3]
```diff
@@ -1,16 +1,17 @@
-# bash programmable completion for zcashd(1)
-# Copyright (c) 2012-2017 The Bitcoin Core developers
+# bash programmable completion for bitcoinzd(1)
+# Copyright (c) 2017-2018 The BitcoinZ developers
```
Comment (renuzit) [inline contrib/bitcoinzd.bash-completion]: And here

**Comment by renuzit** [comment]
Is there any particular reason for renaming the variables starting with 'zcash' inside of BitcoinZ? It seems like such a change would make it more difficult to merge changes from upstream in the future.

File: contrib/bitcoinz-cli.bash-completion (hunk 1 above)
Comment (egyptianbman) [reply contrib/bitcoinz-cli.bash-completion]: The original copyrights are below, Github is just being dumb.

File: contrib/bitcoinz-tx.bash-completion (hunk 2 above)
Comment (egyptianbman) [reply contrib/bitcoinz-tx.bash-completion]: Same here

File: contrib/bitcoinzd.bash-completion (hunk 3 above)
Comment (egyptianbman) [reply contrib/bitcoinzd.bash-completion]: And here

**Comment by egyptianbman** [comment]
To which variables are you referring? I tried to only rename variables that referenced paths, the executables or documentation only.

**Comment by egyptianbman** [comment]
```
[==========] Running 127 tests from 25 test cases.
[----------] Global test environment set-up.
[----------] 24 tests from checktransaction_tests
[ RUN      ] checktransaction_tests.check_vpub_not_both_nonzero
[       OK ] checktransaction_tests.check_vpub_not_both_nonzero (0 ms)
[ RUN      ] checktransaction_tests.valid_transaction
[       OK ] checktransaction_tests.valid_transaction (0 ms)
[ RUN      ] checktransaction_tests.BadVersionTooLow
[       OK ] checktransaction_tests.BadVersionTooLow (1 ms)
[ RUN      ] checktransaction_tests.bad_txns_vin_empty
[       OK ] checktransaction_tests.bad_txns_vin_empty (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vout_empty
[       OK ] checktransaction_tests.bad_txns_vout_empty (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_oversize
[       OK ] checktransaction_tests.bad_txns_oversize (2 ms)
[ RUN      ] checktransaction_tests.bad_txns_vout_negative
[       OK ] checktransaction_tests.bad_txns_vout_negative (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vout_toolarge
[       OK ] checktransaction_tests.bad_txns_vout_toolarge (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_txouttotal_toolarge_outputs
[       OK ] checktransaction_tests.bad_txns_txouttotal_toolarge_outputs (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_txouttotal_toolarge_joinsplit
[       OK ] checktransaction_tests.bad_txns_txouttotal_toolarge_joinsplit (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_txintotal_toolarge_joinsplit
[       OK ] checktransaction_tests.bad_txns_txintotal_toolarge_joinsplit (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vpub_old_negative
[       OK ] checktransaction_tests.bad_txns_vpub_old_negative (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vpub_new_negative
[       OK ] checktransaction_tests.bad_txns_vpub_new_negative (1 ms)
[ RUN      ] checktransaction_tests.bad_txns_vpub_old_toolarge
[       OK ] checktransaction_tests.bad_txns_vpub_old_toolarge (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vpub_new_toolarge
[       OK ] checktransaction_tests.bad_txns_vpub_new_toolarge (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_vpubs_both_nonzero
[       OK ] checktransaction_tests.bad_txns_vpubs_both_nonzero (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_inputs_duplicate
[       OK ] checktransaction_tests.bad_txns_inputs_duplicate (0 ms)
[ RUN      ] checktransaction_tests.bad_joinsplits_nullifiers_duplicate_same_joinsplit
[       OK ] checktransaction_tests.bad_joinsplits_nullifiers_duplicate_same_joinsplit (1 ms)
[ RUN      ] checktransaction_tests.bad_joinsplits_nullifiers_duplicate_different_joinsplit
[       OK ] checktransaction_tests.bad_joinsplits_nullifiers_duplicate_different_joinsplit (0 ms)
[ RUN      ] checktransaction_tests.bad_cb_has_joinsplits
[       OK ] checktransaction_tests.bad_cb_has_joinsplits (0 ms)
[ RUN      ] checktransaction_tests.bad_cb_empty_scriptsig
[       OK ] checktransaction_tests.bad_cb_empty_scriptsig (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_prevout_null
[       OK ] checktransaction_tests.bad_txns_prevout_null (0 ms)
[ RUN      ] checktransaction_tests.bad_txns_invalid_joinsplit_signature
[       OK ] checktransaction_tests.bad_txns_invalid_joinsplit_signature (1 ms)
[ RUN      ] checktransaction_tests.non_canonical_ed25519_signature
[       OK ] checktransaction_tests.non_canonical_ed25519_signature (0 ms)
[----------] 24 tests from checktransaction_tests (7 ms total)

[----------] 5 tests from wallet_zkeys_tests
[ RUN      ] wallet_zkeys_tests.store_and_load_zkeys
[       OK ] wallet_zkeys_tests.store_and_load_zkeys (1 ms)
[ RUN      ] wallet_zkeys_tests.StoreAndLoadViewingKeys
[       OK ] wallet_zkeys_tests.StoreAndLoadViewingKeys (0 ms)
[ RUN      ] wallet_zkeys_tests.write_zkey_direct_to_db
[       OK ] wallet_zkeys_tests.write_zkey_direct_to_db (3 ms)
[ RUN      ] wallet_zkeys_tests.WriteViewingKeyDirectToDB
[       OK ] wallet_zkeys_tests.WriteViewingKeyDirectToDB (1 ms)
[ RUN      ] wallet_zkeys_tests.write_cryptedzkey_direct_to_db
[       OK ] wallet_zkeys_tests.write_cryptedzkey_direct_to_db (429 ms)
[----------] 5 tests from wallet_zkeys_tests (434 ms total)

[----------] 1 test from tautologies
[ RUN      ] tautologies.seven_eq_seven
[       OK ] tautologies.seven_eq_seven (0 ms)
[----------] 1 test from tautologies (0 ms total)

[----------] 9 tests from DeprecationTest
[ RUN      ] DeprecationTest.NonDeprecatedNodeKeepsRunning
[       OK ] DeprecationTest.NonDeprecatedNodeKeepsRunning (0 ms)
[ RUN      ] DeprecationTest.NodeNearDeprecationIsWarned
[       OK ] DeprecationTest.NodeNearDeprecationIsWarned (0 ms)
[ RUN      ] DeprecationTest.NodeNearDeprecationWarningIsNotDuplicated
[       OK ] DeprecationTest.NodeNearDeprecationWarningIsNotDuplicated (0 ms)
[ RUN      ] DeprecationTest.NodeNearDeprecationWarningIsRepeatedOnStartup
[       OK ] DeprecationTest.NodeNearDeprecationWarningIsRepeatedOnStartup (0 ms)
[ RUN      ] DeprecationTest.DeprecatedNodeShutsDown
[       OK ] DeprecationTest.DeprecatedNodeShutsDown (0 ms)
[ RUN      ] DeprecationTest.DeprecatedNodeErrorIsNotDuplicated
[       OK ] DeprecationTest.DeprecatedNodeErrorIsNotDuplicated (0 ms)
[ RUN      ] DeprecationTest.DeprecatedNodeErrorIsRepeatedOnStartup
[       OK ] DeprecationTest.DeprecatedNodeErrorIsRepeatedOnStartup (0 ms)
[ RUN      ] DeprecationTest.DeprecatedNodeShutsDownIfOldVersionDisabled
[       OK ] DeprecationTest.DeprecatedNodeShutsDownIfOldVersionDisabled (0 ms)
[ RUN      ] DeprecationTest.DeprecatedNodeKeepsRunningIfCurrentVersionDisabled
[       OK ] DeprecationTest.DeprecatedNodeKeepsRunningIfCurrentVersionDisabled (0 ms)
[----------] 9 tests from DeprecationTest (0 ms total)

[----------] 5 tests from equihash_tests
[ RUN      ] equihash_tests.expand_and_contract_arrays
[       OK ] equihash_tests.expand_and_contract_arrays (0 ms)
[ RUN      ] equihash_tests.minimal_solution_representation
[       OK ] equihash_tests.minimal_solution_representation (0 ms)
[ RUN      ] equihash_tests.is_probably_duplicate
[       OK ] equihash_tests.is_probably_duplicate (0 ms)
[ RUN      ] equihash_tests.check_basic_solver_cancelled
[       OK ] equihash_tests.check_basic_solver_cancelled (3 ms)
[ RUN      ] equihash_tests.check_optimised_solver_cancelled
[       OK ] equihash_tests.check_optimised_solver_cancelled (2 ms)
[----------] 5 tests from equihash_tests (5 ms total)

[----------] 3 tests from HTTPRPC
[ RUN      ] HTTPRPC.FailsOnGET
[       OK ] HTTPRPC.FailsOnGET (0 ms)
[ RUN      ] HTTPRPC.FailsWithoutAuthHeader
[       OK ] HTTPRPC.FailsWithoutAuthHeader (0 ms)
[ RUN      ] HTTPRPC.FailsWithBadAuth
[       OK ] HTTPRPC.FailsWithBadAuth (250 ms)
[----------] 3 tests from HTTPRPC (250 ms total)

[----------] 3 tests from joinsplit
[ RUN      ] joinsplit.h_sig
[       OK ] joinsplit.h_sig (0 ms)
[ RUN      ] joinsplit.full_api_test
[       OK ] joinsplit.full_api_test (55185 ms)
[ RUN      ] joinsplit.note_plaintexts
[       OK ] joinsplit.note_plaintexts (0 ms)
[----------] 3 tests from joinsplit (55185 ms total)

[----------] 4 tests from keystore_tests
[ RUN      ] keystore_tests.store_and_retrieve_spending_key
[       OK ] keystore_tests.store_and_retrieve_spending_key (0 ms)
[ RUN      ] keystore_tests.store_and_retrieve_note_decryptor
[       OK ] keystore_tests.store_and_retrieve_note_decryptor (0 ms)
[ RUN      ] keystore_tests.StoreAndRetrieveViewingKey
[       OK ] keystore_tests.StoreAndRetrieveViewingKey (1 ms)
[ RUN      ] keystore_tests.store_and_retrieve_spending_key_in_encrypted_store
[       OK ] keystore_tests.store_and_retrieve_spending_key_in_encrypted_store (0 ms)
[----------] 4 tests from keystore_tests (1 ms total)

[----------] 6 tests from noteencryption
[ RUN      ] noteencryption.api
[       OK ] noteencryption.api (122 ms)
[ RUN      ] noteencryption.prf_addr
[       OK ] noteencryption.prf_addr (0 ms)
[ RUN      ] noteencryption.prf_nf
[       OK ] noteencryption.prf_nf (0 ms)
[ RUN      ] noteencryption.prf_pk
[       OK ] noteencryption.prf_pk (1 ms)
[ RUN      ] noteencryption.prf_rho
[       OK ] noteencryption.prf_rho (0 ms)
[ RUN      ] noteencryption.uint252
[       OK ] noteencryption.uint252 (0 ms)
[----------] 6 tests from noteencryption (123 ms total)

[----------] 2 tests from Mempool
[ RUN      ] Mempool.PriorityStatsDoNotCrash
[       OK ] Mempool.PriorityStatsDoNotCrash (32 ms)
[ RUN      ] Mempool.TxInputLimit
[       OK ] Mempool.TxInputLimit (0 ms)
[----------] 2 tests from Mempool (32 ms total)

[----------] 8 tests from merkletree
[ RUN      ] merkletree.vectors
[       OK ] merkletree.vectors (24549 ms)
[ RUN      ] merkletree.emptyroots
[       OK ] merkletree.emptyroots (0 ms)
[ RUN      ] merkletree.emptyroot
[       OK ] merkletree.emptyroot (0 ms)
[ RUN      ] merkletree.deserializeInvalid
[       OK ] merkletree.deserializeInvalid (0 ms)
[ RUN      ] merkletree.deserializeInvalid2
[       OK ] merkletree.deserializeInvalid2 (0 ms)
[ RUN      ] merkletree.deserializeInvalid3
[       OK ] merkletree.deserializeInvalid3 (0 ms)
[ RUN      ] merkletree.deserializeInvalid4
[       OK ] merkletree.deserializeInvalid4 (0 ms)
[ RUN      ] merkletree.testZeroElements
[       OK ] merkletree.testZeroElements (1 ms)
[----------] 8 tests from merkletree (24550 ms total)

[----------] 1 test from Miner
[ RUN      ] Miner.GetMinerScriptPubKey
[       OK ] Miner.GetMinerScriptPubKey (0 ms)
[----------] 1 test from Miner (0 ms total)

[----------] 1 test from PoW
[ RUN      ] PoW.DifficultyAveraging
[       OK ] PoW.DifficultyAveraging (0 ms)
[----------] 1 test from PoW (0 ms total)

[----------] 1 test from Random
[ RUN      ] Random.MappedShuffle
[       OK ] Random.MappedShuffle (0 ms)
[----------] 1 test from Random (0 ms total)

[----------] 1 test from rpc
[ RUN      ] rpc.check_blockToJSON_returns_minified_solution
[       OK ] rpc.check_blockToJSON_returns_minified_solution (0 ms)
[----------] 1 test from rpc (0 ms total)

[----------] 1 test from Transaction
[ RUN      ] Transaction.JSDescriptionRandomized
[       OK ] Transaction.JSDescriptionRandomized (1 ms)
[----------] 1 test from Transaction (1 ms total)

[----------] 2 tests from Validation
[ RUN      ] Validation.ContextualCheckInputsPassesWithCoinbase
[       OK ] Validation.ContextualCheckInputsPassesWithCoinbase (0 ms)
[ RUN      ] Validation.ReceivedBlockTransactions
[       OK ] Validation.ReceivedBlockTransactions (0 ms)
[----------] 2 tests from Validation (0 ms total)

[----------] 3 tests from circuit
[ RUN      ] circuit.values
[       OK ] circuit.values (1 ms)
[ RUN      ] circuit.endianness
[       OK ] circuit.endianness (0 ms)
[ RUN      ] circuit.merkle_tree_gadget_weirdness
[       OK ] circuit.merkle_tree_gadget_weirdness (14492 ms)
[----------] 3 tests from circuit (14493 ms total)

[----------] 2 tests from txid_tests
[ RUN      ] txid_tests.check_txid_and_hash_are_same
[       OK ] txid_tests.check_txid_and_hash_are_same (0 ms)
[ RUN      ] txid_tests.check_txid_and_hash_are_same_coinbase
[       OK ] txid_tests.check_txid_and_hash_are_same_coinbase (0 ms)
[----------] 2 tests from txid_tests (0 ms total)

[----------] 2 tests from libzcash_utils
[ RUN      ] libzcash_utils.convertBytesVectorToVector
[       OK ] libzcash_utils.convertBytesVectorToVector (0 ms)
[ RUN      ] libzcash_utils.convertVectorToInt
[       OK ] libzcash_utils.convertVectorToInt (0 ms)
[----------] 2 tests from libzcash_utils (0 ms total)

[----------] 19 tests from proofs
[ RUN      ] proofs.g1_pairing_at_infinity
[       OK ] proofs.g1_pairing_at_infinity (1672 ms)
[ RUN      ] proofs.g2_subgroup_check
[       OK ] proofs.g2_subgroup_check (12 ms)
[ RUN      ] proofs.sqrt_zero
[       OK ] proofs.sqrt_zero (0 ms)
[ RUN      ] proofs.sqrt_fq
[       OK ] proofs.sqrt_fq (36 ms)
[ RUN      ] proofs.sqrt_fq2
[       OK ] proofs.sqrt_fq2 (388 ms)
[ RUN      ] proofs.size_is_expected
[       OK ] proofs.size_is_expected (0 ms)
[ RUN      ] proofs.fq_serializes_properly
[       OK ] proofs.fq_serializes_properly (1 ms)
[ RUN      ] proofs.fq2_serializes_properly
[       OK ] proofs.fq2_serializes_properly (1 ms)
[ RUN      ] proofs.fq_valid
[       OK ] proofs.fq_valid (0 ms)
[ RUN      ] proofs.fq_invalid
[       OK ] proofs.fq_invalid (0 ms)
[ RUN      ] proofs.fq2_valid
[       OK ] proofs.fq2_valid (0 ms)
[ RUN      ] proofs.fq2_invalid
[       OK ] proofs.fq2_invalid (0 ms)
[ RUN      ] proofs.g1_serializes_properly
[       OK ] proofs.g1_serializes_properly (200 ms)
[ RUN      ] proofs.g2_serializes_properly
[       OK ] proofs.g2_serializes_properly (1982 ms)
[ RUN      ] proofs.zksnark_serializes_properly
Swap is not beneficial, not performing
* QAP number of variables: 252
* QAP pre degree: 250
* QAP degree: 256
* QAP number of input variables: 4
* G1 window: 8
* G2 window: 6
* G1 elements in PK: 1783
* Non-zero G1 elements in PK: 1763
* G2 elements in PK: 254
* Non-zero G2 elements in PK: 252
* PK size in bits: 625897
* G1 elements in VK: 6
* G2 elements in VK: 5
* VK size in bits: 4586
[       OK ] proofs.zksnark_serializes_properly (3597 ms)
[ RUN      ] proofs.g1_deserialization
[       OK ] proofs.g1_deserialization (78 ms)
[ RUN      ] proofs.g2_deserialization
[       OK ] proofs.g2_deserialization (2760 ms)
[ RUN      ] proofs.g1_test_vectors
[       OK ] proofs.g1_test_vectors (2116 ms)
[ RUN      ] proofs.g2_test_vectors
[       OK ] proofs.g2_test_vectors (18553 ms)
[----------] 19 tests from proofs (31396 ms total)

[----------] 1 test from paymentdisclosure
[ RUN      ] paymentdisclosure.mainnet
Test payment disclosure database created in folder: /tmp/f8d0-42e1-e61d-e846
[       OK ] paymentdisclosure.mainnet (3466 ms)
[----------] 1 test from paymentdisclosure (3466 ms total)

[----------] 1 test from CheckBlock
[ RUN      ] CheckBlock.VersionTooLow
[       OK ] CheckBlock.VersionTooLow (0 ms)
[----------] 1 test from CheckBlock (0 ms total)

[----------] 1 test from ContextualCheckBlock
[ RUN      ] ContextualCheckBlock.BadCoinbaseHeight
[       OK ] ContextualCheckBlock.BadCoinbaseHeight (0 ms)
[----------] 1 test from ContextualCheckBlock (0 ms total)

[----------] 21 tests from wallet_tests
[ RUN      ] wallet_tests.setup_datadir_location_run_as_first_test
[       OK ] wallet_tests.setup_datadir_location_run_as_first_test (0 ms)
[ RUN      ] wallet_tests.note_data_serialisation
[       OK ] wallet_tests.note_data_serialisation (1 ms)
[ RUN      ] wallet_tests.find_unspent_notes
[       OK ] wallet_tests.find_unspent_notes (1 ms)
[ RUN      ] wallet_tests.set_note_addrs_in_cwallettx
[       OK ] wallet_tests.set_note_addrs_in_cwallettx (1 ms)
[ RUN      ] wallet_tests.set_invalid_note_addrs_in_cwallettx
[       OK ] wallet_tests.set_invalid_note_addrs_in_cwallettx (0 ms)
[ RUN      ] wallet_tests.GetNoteNullifier
[       OK ] wallet_tests.GetNoteNullifier (1 ms)
[ RUN      ] wallet_tests.FindMyNotes
[       OK ] wallet_tests.FindMyNotes (0 ms)
[ RUN      ] wallet_tests.FindMyNotesInEncryptedWallet
[       OK ] wallet_tests.FindMyNotesInEncryptedWallet (1 ms)
[ RUN      ] wallet_tests.get_conflicted_notes
[       OK ] wallet_tests.get_conflicted_notes (1 ms)
[ RUN      ] wallet_tests.nullifier_is_spent
[       OK ] wallet_tests.nullifier_is_spent (1 ms)
[ RUN      ] wallet_tests.navigate_from_nullifier_to_note
[       OK ] wallet_tests.navigate_from_nullifier_to_note (1 ms)
[ RUN      ] wallet_tests.spent_note_is_from_me
[       OK ] wallet_tests.spent_note_is_from_me (1 ms)
[ RUN      ] wallet_tests.cached_witnesses_empty_chain
[       OK ] wallet_tests.cached_witnesses_empty_chain (1095 ms)
[ RUN      ] wallet_tests.cached_witnesses_chain_tip
[       OK ] wallet_tests.cached_witnesses_chain_tip (1 ms)
[ RUN      ] wallet_tests.CachedWitnessesDecrementFirst
[       OK ] wallet_tests.CachedWitnessesDecrementFirst (2 ms)
[ RUN      ] wallet_tests.CachedWitnessesCleanIndex
[       OK ] wallet_tests.CachedWitnessesCleanIndex (230 ms)
[ RUN      ] wallet_tests.ClearNoteWitnessCache
[       OK ] wallet_tests.ClearNoteWitnessCache (1 ms)
[ RUN      ] wallet_tests.WriteWitnessCache
[       OK ] wallet_tests.WriteWitnessCache (1 ms)
[ RUN      ] wallet_tests.UpdateNullifierNoteMap
[       OK ] wallet_tests.UpdateNullifierNoteMap (0 ms)
[ RUN      ] wallet_tests.UpdatedNoteData
[       OK ] wallet_tests.UpdatedNoteData (1 ms)
[ RUN      ] wallet_tests.MarkAffectedTransactionsDirty
[       OK ] wallet_tests.MarkAffectedTransactionsDirty (1 ms)
[----------] 21 tests from wallet_tests (1341 ms total)

[----------] Global test environment tear-down
[==========] 127 tests from 25 test cases ran. (131285 ms total)
[  PASSED  ] 127 tests.

  YOU HAVE 1 DISABLED TEST
```

File: src/util.cpp [hunk 4]
```diff
@@ -901,7 +901,7 @@ std::string LicenseInfo()
            FormatParagraph(strprintf(_("Copyright (C) 2009-%i The Bitcoin Core Developers"), COPYRIGHT_YEAR)) + "\n" +
            FormatParagraph(strprintf(_("Copyright (C) 2015-%i The Zcash Developers"), COPYRIGHT_YEAR)) + "\n" +
            FormatParagraph(strprintf(_("Copyright (C) 2015-%i zdeveloper.org"), COPYRIGHT_YEAR)) + "\n" +
-           FormatParagraph(strprintf(_("Copyright (C) 2017-%i BitcoinZ team"), COPYRIGHT_YEAR)) + "\n" +
+           FormatParagraph(strprintf(_("Copyright (C) 2017-%i BitcoinZ Core Developers"), COPYRIGHT_YEAR)) + "\n" +
```
Comment (renuzit) [inline src/util.cpp:904]: Leave credit for BitcoinZ team

Comment (egyptianbman) [reply src/util.cpp:904]: I was trying to be consistent with other areas: https://github.com/btcz/bitcoinz/pull/42/files#diff-b9f4ea3882f3685ed32cfb7b663cdec3R1 -- but I failed to do so.

So should I update everything to:
- BitcoinZ Team
- BitcoinZ Developers
- BitcoinZ Core Developers

It also looks like other teams have "The" in front.

I don't care which we choose.

Comment (renuzit) [reply src/util.cpp:904]: 🥇

**Review by renuzit — DISMISSED** [review]

**Review by renuzit — DISMISSED** [review]

**Review by renuzit — COMMENTED** [review]
Need to update CLIENT_VERSION_MINOR CLIENT_VERSION_REVISION and the following files:
./configure.ac
./src/clientversion.h

**Review by renuzit — APPROVED** [review]

**Comment by egyptianbman** [comment]
Thanks!

**Pull request merged**